packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_distribution_repository/__brick__/android/keyAlias.txt

```diff
@@ -0,0 +1 @@
+REPLACE_THIS_WITH_YOUR_KEY_ALIAS
```

Comment:
Shouldn't we have files for the other extensions as well, such as auth_key.p8 ? I guess this will be more obvious to the developers what are the expected files they need to provide.

Reply:
Sure, we can have those, but they won't serve any purpose as they will be over-written by the new ones. On the contrary, the generated `.txt` files are created so the user doesn't have to do it manually (and thus accidentally making a mistake in naming them).

But in case you'd like me to create those, which files do you want me to include? Even the `android.jks` and empty provisioning profiles?

Reply:
The purpose of adding all files, including the non-txt files, is to reduce the guess work as much as possible by indicating all expected files that the repository need to contain. This will be nice compliment to the documentation. They should be empty and the developers should replace them with a real ones.